proposals/0000-linear-types.rst

```diff
@@ -962,7 +999,7 @@ example:
   data Socket (n :: *) (s :: State)
   data Closed (n :: *)
 
-  socket :: RIO (forall n. Socket n 'Unbound :'A-> RIO (Unrestricted a, Closed s)) :'A -> RIO (Unrestricted a)
+  newSocket :: RIO (forall n. Socket n 'Unbound :'A-> RIO (Unrestricted a, Closed s)) :'A -> RIO (Unrestricted a)
```

Comment:
Note that I called it `socket` because it's [the name](https://www.stackage.org/haddock/lts-9.18/network-2.6.3.2/Network-Socket.html#v:socket) of this primitive in the socket library.